### ISSUE TITLE
Minor Facebook Request

### ISSUE THREAD

**Issue by blackearth2014** [mschack/VCV-Rack-Plugins#35]
MSHack,

Some Mod Artists asked if you could add gate outputs to your sequencers? Right now now they only have trigger outputs.

Now I was thinking that the feature that says "no trigger" seconds as the gate output right?

**Comment by mschack** [comment]
By gate do you mean a value that is held for the duration  of the note?  No trigger, if you are talking about the Triad  Sequencer, will output the note but not give a trigger.  Kinda like legato, which is probably what I should have called it.

**Comment by blackearth2014** [comment]
Yes, I would assume so. I am just passing the request along, but for what I am using the Triad Sequencer, it sounds about right.

This is your season
Omar Brown
Blessed House Media

On Dec 11, 2017, at 12:21 PM, mschack <notifications@github.com> wrote:

By gate do you mean a value that is held for the duration of the note? No trigger, if you are talking about the Triad Sequencer, will output the note but not give a trigger. Kinda like legato, which is probably what I should have called it.
